context/packages.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+if [ "$EXTRA_APT_PACKAGES" ]; then
+    if ! grep -i "ubuntu" /etc/os-release > /dev/null; then
+      echo "EXTRA_APT_PACKAGES var provided in a non-Ubuntu environment."
+      echo "Exiting..."
+      exit 1
+    fi
+    echo "EXTRA_APT_PACKAGES environment variable found. Installing packages."
+    apt update -y
+    apt install -y --no-install-recommends $EXTRA_APT_PACKAGES
+fi
+
+if [ "$EXTRA_YUM_PACKAGES" ]; then
+    if ! grep -i "centos" /etc/os-release > /dev/null; then
+      echo "EXTRA_YUM_PACKAGES var provided in a non-Centos environment."
+      echo "Exiting..."
+      exit 1
+    fi
+    echo "EXTRA_YUM_PACKAGES environment variable found. Installing packages."
+    yum check-update -y
+    yum install -y $EXTRA_YUM_PACKAGES
+fi
+
+if [ -e "/opt/rapids/environment.yml" ]; then
+    echo "environment.yml found. Installing packages"
+    conda env update -f /opt/rapids/environment.yml
+fi
+
+if [ "$EXTRA_CONDA_PACKAGES" ]; then
+    echo "EXTRA_CONDA_PACKAGES environment variable found. Installing packages."
+    conda install -y $EXTRA_CONDA_PACKAGES
```

Comment:
Just checking is the rapids environment already activated here?

Reply:
yup! that's correct. it gets run after a `conda activate rapids` call [here](https://github.com/ajschmidt8/docker/blob/user-pkgs/context/entrypoint.sh#L17-L20)